src/__Private/Resolution/get_current_namespace.hack

```diff
@@ -13,20 +13,12 @@ use type Facebook\HHAST\{Node, Script};
 use namespace HH\Lib\C;
 
 function get_current_namespace(Script $root, Node $node): ?string {
-  $namespaces = $root->getNamespaces();
-  if (C\is_empty($namespaces)) {
-    return null;
-  }
-  if ($namespaces[0]['statement']) {
-    return $namespaces[0]['decl']->getQualifiedNameAsString();
-  }
```

Comment:
I added the global namespace to the returned list so that we wouldn't need to handle these special cases everywhere (also this code wouldn't be correct for files with bodyless namespaces).